components/Blog/blog.scss

```diff
@@ -0,0 +1,68 @@
+.blog-page {
+  display: flex;
+}
+
+.side-nav-blog {
+  float: left;
+  align-self: center;
+
+  a {
+    display: block;
+    padding: 15px 25px 15px 100%;
```

Comment:
Just a heads up, we are using `rem` in place of `px` and `em`

Reply:
This padding is currently messing up the nav bar.

I'm suggesting something like `pa2` or `pt2 ph3` using Tachyons on the element and then centering the text.